src/strings/en-us.json

```diff
@@ -144,7 +144,9 @@
     "ChannelNameOnly": "Channel {0} only",
     "ChannelNumber": "Channel number",
     "Channels": "Channels",
-    "CinemaModeConfigurationHelp": "Cinema mode brings the theater experience straight to your living room with the ability to play trailers and custom intros before the main feature.",
+    "CinemaModeConfigurationHelp": "Cinema mode brings the theater experience straight to your living room with the ability to play trailers before the main feature.",
```

Comment:
I couldn't find the feature to play custom intros, so I removed this part from the translation.
Does this feature exist?

Reply:
It isn’t implemented in the server and is pending removal https://github.com/jellyfin/jellyfin/pull/2428